pkg/process/process_unix.go

```diff
@@ -57,25 +57,25 @@ func Kill(pid int) error {
 	return nil
 }
 
-// Zombie return true if process has a state with "Z". It only considers positive
+// Zombie return "Z" if process has a state with "Z". It only considers positive
 // PIDs; 0 (all processes in the current process group), -1 (all processes with
 // a PID larger than 1), and negative (-n, all processes in process group "n")
 // values for pid are ignored. Refer to [PROC(5)] for details.
 //
 // [PROC(5)]: https://man7.org/linux/man-pages/man5/proc.5.html
-func Zombie(pid int) (bool, error) {
+func GetProcStatus(pid int) (string, error) {
```

Comment:
- `Zombie` is an exported function, so we can't just remove it 😅 (it could be a wrapper for `GetProcStatus() == "Z")`
- Perhaps the function should be named `Status()` (it would be called as `process.Status(pid)`

Reply:
^^ edited; `State()` would be a better name, to reflect the field name in `proc/<pid>/stat`